tensorflow/core/kernels/mlir_generated/build_defs.bzl

```diff
@@ -269,17 +277,31 @@ def _gen_kernel_library(
         jit_types = []
     if not output_jit_types:
         output_jit_types = jit_types
-
     true_jits = [True for i in range(len(jit_types))]
-    all_jit_kernels = zip(jit_types, output_jit_types, true_jits)
+    if not jit_i64_indexed_for_large_tensors_types:
+        jit_i64_indexed_for_large_tensors_types = []
+    if not output_partial_jit_types:
+        output_partial_jit_types = jit_i64_indexed_for_large_tensors_types
+    true_partial_jits = [True for i in range(
+        len(jit_i64_indexed_for_large_tensors_types))]
+    use_i64_indices_for_jit = [False for i in range(len(jit_types))]
+    use_i64_indices_for_partial_jit = [True for i in range(
+        len(jit_i64_indexed_for_large_tensors_types))]
+    all_jit_kernels = zip(jit_types, output_jit_types,
+                          true_jits, use_i64_indices_for_jit)
+    all_partial_jit_kernels = zip(jit_i64_indexed_for_large_tensors_types,
```

Comment:
For consistency, maybe call this `all_jit_i64_indexed_for_large_tensors_kernels`

Reply:
Name changed